torch/csrc/jit/mobile/flatbuffer_loader.cpp

```diff
@@ -368,14 +375,14 @@ IValue parseIntList(
   return parseListNative<int64_t>(list);
 }
 
-IValue parseBoolList(
+IValue parseDoubleList(
```

Comment:
OOC, why wasn't this caught by tests?

Reply:
Good point! Added a test to catch this case.

Reply:
Awesome! Could we also proactively check for code coverage to make sure there are no missed coverages elsewhere?